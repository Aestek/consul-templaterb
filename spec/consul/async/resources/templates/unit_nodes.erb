Nodes
<%
  nodes().each do |node|
%><%= node['Node'] %>
<%
  end
%>