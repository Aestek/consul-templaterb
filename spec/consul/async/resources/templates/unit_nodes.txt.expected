Nodes
consul01-par.central.criteo.preprod
consul02-par.central.criteo.preprod
consul03-par.central.criteo.preprod
